src/main/views/macro/date.njk

```diff
@@ -11,19 +11,33 @@
     {% if monthError %}<span class="error-message">{{ t(monthError) }}</span>{% endif %}
     {% if yearError %}<span class="error-message">{{ t(yearError) }}</span>{% endif %}
     {% if dateError %}<span class="error-message">{{ t(dateError) }}</span>{% endif %}
+    <fieldset id="dateofbirth[label]">
     <div class="form-date" id="{{ name }}[label]">
       <div class="form-group form-group-day">
+        <legend> 
+          <span class="visuallyhidden">
+          Enter Day of your date of birth</span>
+      </legend>
         <label class="form-label" id="{{ name }}[day][label]" for="{{ name }}[day]">{{ t('Day') }}</label>
         <input class="form-control {% if dayError or dateError %} form-control-error {% endif %}" id="{{ name }}[day]" name="{{ name }}[day]" value="{{ form.valueFor(name).day | default('') }}" type="number" aria-describedby="dob-hint">
       </div>
       <div class="form-group form-group-month">
+        <legend> 
```

Comment:
Ideally only one legend is allowed i believe. This needs to be addressed

Reply:
Yes, actually we no need to keep **visuallyhidden** inside the legend, if this not intentional it's good to remove them.